cmake/detect-intrinsics.cmake

```diff
@@ -2,8 +2,12 @@
 # Licensed under the Zlib license, see LICENSE.md for details
 
 macro(check_acle_compiler_flag)
-    if(NOT NATIVEFLAG AND NOT HAVE_ACLE_FLAG)
-        set(ACLEFLAG "-march=armv8-a+crc" CACHE INTERNAL "Compiler option to enable ACLE support")
+    if(CMAKE_C_COMPILER_ID MATCHES "GNU" OR CMAKE_C_COMPILER_ID MATCHES "Clang")
+        if(NOT NATIVEFLAG AND NOT HAVE_ACLE_FLAG)
+            set(ACLEFLAG "-march=armv8-a+crc" CACHE INTERNAL "Compiler option to enable ACLE support")
+        endif()
+    elseif(MSVC)
```

Comment:
Should this only be set if ARM64?

Reply:
I have already reported the missing ACLE intrinsics for 32-bit ARM targets to Microsoft... I'm not sure yet if they will support 32-bit ARMv8 or not...